drivers/sensor/memsic/mc3419/mc3419.c

```diff
@@ -16,46 +16,39 @@
 LOG_MODULE_REGISTER(MC3419, CONFIG_SENSOR_LOG_LEVEL);
 
 static const uint16_t mc3419_accel_sense_map[] = {1, 2, 4, 8, 6};
-static struct mc3419_odr_map odr_map_table[] = {
-	{25}, {50}, {62, 500}, {100},
-	{125}, {250}, {500}, {1000}
-};
+static struct mc3419_odr_map odr_map_table[] = {{25},  {50},  {62, 500}, {100},
+						{125}, {250}, {500},     {1000}};
```

Comment:
Please limit the commit `drivers: sensors: move mc3419 into parent memsic folder` to moving files only. Don't make clang formatting changes too.

Reply:
One of the workflows failed complaining I was introducing code that wasn't clang formatted when I moved the files, so I did. I've undone it now, hopefully you can override the workflow if it fails again.

Reply:
clang-format warnings are noisy, but there are some real things buried in there:
- The keep-sorted directive is highlighting that the memsic line is in the wrong place in the cmake and Kconfig files.
- There is a long line of 106 columns in drivers/sensor/memsic/mmc56x3/mmc56x3.c

Fix those and I don't think a workflow override will be needed.

Reply:
My bad, I should've actually read them :sweat_smile: Updated to fix those

Reply:
The cmake and Kconfig fixes should be in the first commit, not the second.

Reply:
Sorry, moved to first commit